exercises/container-security/docker-hub-cli/README.md

```diff
@@ -0,0 +1,41 @@
+# Docker Hub CLI exercise
+
+CLI to use Docker API with Docker Hub registry.
+
+
+## Prerequisites
+
+Node 6 or above.
+
+## Usage
+
+Clone the repo locally.
```

Comment:
Perhaps also mention `npm install`? Should be obvious, I know :)

Reply:
Yes, thought of that, good point, I'll add it.